<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SongBook</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SongBookResources.qrc">
    <normaloff>:/kuvia/musical_note_nicu_bucule_01.png</normaloff>:/kuvia/musical_note_nicu_bucule_01.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 223, 223);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="viewerText">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Lyrics Viewer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="browserText">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Lyrics Browser</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="lyricsViewer">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
       </font>
      </property>
      <property name="toolTip">
       <string>Click here to browse for .txt files on your device.</string>
      </property>
      <property name="text">
       <string>Browse for lyrics</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="lyricsBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SongBookResources.qrc"/>
 </resources>
 <connections/>
</ui>
